plugins/executor/plugins.go

```diff
@@ -8,6 +8,20 @@ import (
 	"github.com/hashicorp/go-plugin"
 )
 
+// ExecutorConfig is the config that Nomad passes to the executor
+type ExecutorConfig struct {
+
+	// LogFile is the file to which Executor logs
+	LogFile string
+
+	// LogLevel is the level of the logs to putout
+	LogLevel string
+
+	// FSIsolation if set will use an executor implementation that support
+	// filesystem isolation
+	FSIsolation bool
+}
+
 var HandshakeConfig = plugin.HandshakeConfig{
```

Comment:
While you're here lets remove this and use `base.Handshake`.

Reply:
Done